examples/undocumented/python_modular/graphical/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+INCLUDE(PythonEnvironment)
+GET_PYTHON_ENV()
+
+# FILE(GLOB PYTHON_EXAMPLES *.py)
```

Comment:
We can uncomment this to build all the graphical examples but work needs to be done on the graphical examples to make them build without error. Currently only `classifier_gaussian_process_binary_classification.py` is tested.